Modules/_ssl.c

```diff
@@ -74,6 +74,10 @@ static PySocketModule_APIObject PySocketModule;
 #  endif
 #endif
 
+#if OPENSSL_VERSION_NUMBER < 0x1000200fL
```

Comment:
Is this still needed after the other change to require 1.0.2?

Reply:
Not really, I added it for doubleplusgood paranoia reasons and just removed it.